syndat/sample_levels.py

```diff
@@ -59,31 +59,41 @@ def wigner_PDF(x, avg_level_spacing):
 
 def sample_resonance_levels(E0, N_levels, avg_level_spacing, method):
     
+    E0 = E0+avg_level_spacing*np.random.default_rng().uniform(low=0.0,high=1.0) # offset starting point so we are not just finding the distribution each time
     if method == 'invCDF':
         level_spacing = avg_level_spacing*sample_wigner_invCDF(N_levels)
+        levels = [E0+level_spacing[0]]
+        
+        for ilevel in range(1,N_levels):
+            levels.append(levels[ilevel-1]+level_spacing[ilevel])
             
     elif method == 'GOE':
-        level_spacing = []
-        for ilevel in range(N_levels):
-            X = generate_GOE(2)
-            eigenvalues = np.linalg.eigvals(X)
-            spacing = avg_level_spacing*abs(np.diff(eigenvalues))/(np.pi/2)
-            level_spacing.append(spacing.item())
+        # Use resonance_generator from FUDGE:
+        import sys
+        sys.path.insert(0, '/Users/colefritsch/ENCORE/syndat/nuc_syndat/fudge/')
```

Comment:
We need to find a better solution for user-specific path requirements. I have a similar challenge when trying to wrap around SAMMY. I will look into a way to set these as variables during the build.